products/ocp4/profiles/default.profile

```diff
@@ -45,7 +45,7 @@ selections:
     - api_server_api_priority_v1_flowschema_catch_all
     - file_groupowner_openvswitch
     - gcp_disk_encryption_enabled
-    - kubelet_configure_tls_cipher_suites_ingresscontroller
```

Comment:
There is no need to keep this rule in `default.profile`, since it is selected in CIS Control file.

Reply:
The `default.profile` is not an actual profile, it is a list of rules not selected in any profiles that should be kept in the data stream.